hw5/tasks/task2.go

```diff
@@ -0,0 +1,55 @@
+package tasks
+
+import (
+	"errors"
+	"math"
+)
+
+// MyStrToInt2 converts string to int
+func MyStrToInt2(s string) (int, error) {
```

Comment:
Whoa! This is not a second solution I could think about. Thank you for it, nice.

Reply:
OMG )